### ISSUE TITLE
All Pages - Missing Image ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#14]
While testing using the DHS Trusted Tester process, I found that images are missing text/ALT text to convey the meaning of the image such as the WORKER.gov logo and the drop down arrow for the INPUT field, along with all the little decorative images.

Section 508 - 22(a)
WCAG 2.0 - 2.2 Web: Images

**Comment by jfc3-dol** [comment]
Here are two screenshots from the "All Rights" page - http://worker.gov/profiles/all.html that are missing ALT. I assume there are other pages where icons/images like this are having the same issue.

Part 1

![worker dot gov - images missing alt text - part 1](https://cloud.githubusercontent.com/assets/10423487/19776091/c4e9c22e-9c40-11e6-8fe5-8437b3952d8c.png)

Part 2

![worker dot gov - images missing alt text - part 2](https://cloud.githubusercontent.com/assets/10423487/19776103/ccaf221a-9c40-11e6-9ca3-9ddbc76c7bbd.png)
